src/Microsoft.AspNet.OData.Shared/DefaultEdmODataAPIHandler.cs

```diff
@@ -36,6 +36,8 @@ public DefaultEdmODataAPIHandler(ICollection<IEdmStructuredObject> originalList,
         }
 
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1021:AvoidOutParameters", MessageId = "2#")]
```

Comment:
can these be added to the suppression file?

Reply:
Done